tardis/atomic.py

```diff
@@ -463,11 +464,14 @@ def prepare_atom_data(self, selected_atomic_numbers, line_interaction_type='scat
             maximum ion number to be included in the calculation
 
         """
+        if not self.prepared:
+            self.prepared = True
+        else:
+            raise "AtomData was already prepared"
```

Comment:
One should not raise generic Exceptions. Either raise one of the built-in exceptions (not sure if `ValueError` is descriptive) or better yet make your own Exception by subclassing from Exception or one of its subclasses e.g.:

```python
class AtomDataNotPreparedError(Exception):
    pass
```

and then `raise that`